provisioning/transport/amqp/src/AmqpClientLink.cs

```diff
@@ -40,7 +40,7 @@ public AmqpClientLink(AmqpClientSession amqpClientSession)
 
         public async Task OpenAsync(TimeSpan timeout)
         {
-            if (Extensions.IsReceiver(AmqpLinkSettings))
+            if (Amqp.Extensions.IsReceiver(AmqpLinkSettings))
```

Comment:
Now that the provisioning client uses `.Extensions` namespace, this call was ambiguous.